main.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask,render_template,request,redirect
```

Comment:
This is also not required I guess

Reply:
These are the libraries used.
These are pretty much required. I don't understand

Reply:
I am not talking of the imports. I am saying about this file itself, there is already a main.py under hackerBoy as hackerBoy007/main.py. That is the main.py your are using for your task if I am not wrong. So what is the purpose of this one?